### ISSUE TITLE
Searching for Friends OTH

### ISSUE THREAD

**Issue by Dizzy611** [Insidious611/DancingMadFF6#44]
This has to be my favorite song on the original OST, it's a great world theme, but sadly we're defaulting to OST for it atm due to the OCR version just not sounding right. I want help finding another version of this track to put in to show off the MSU-1 and its capabilities better than just using the CD version of the OST, but that keeps the right genre and feel.

**Comment by edale2** [comment]
Take your pick, I should be able to find better quality copies of most of these:

https://www.youtube.com/watch?v=T5LRq1YkWsw
https://www.youtube.com/watch?v=RcFXiYn6tGc&index=57&list=PLBdL0iJj-imYL0i1kzPXHwuLs5w5-YXXn
https://www.youtube.com/watch?v=zAYZgA5eY7I&feature=youtu.be&t=61
https://www.youtube.com/watch?v=6SzUd2i0j8w
https://www.youtube.com/watch?v=B34aossrdPw
https://www.youtube.com/watch?v=GRRue26mwgQ
https://www.youtube.com/watch?v=98-8wtjOjPU&t=82s
https://www.youtube.com/watch?v=Qf6gVPqrao8
https://www.youtube.com/watch?v=dmHfcr858og
https://www.youtube.com/watch?v=T9O7t7_zoZ4
https://www.youtube.com/watch?v=Nk-bpikgTGs
https://www.youtube.com/watch?v=qB2zxLZli3c


Can't wait until this one gets released on an album, so I can get a clean version of this (First appearance of this in one of their concerts was literally just a few days ago, so it should be on their next album, which would make it... the 5th album I believe?):
https://www.youtube.com/watch?v=9z5cht1hYVQ

And because I know how much you LOVE vocals:
https://www.youtube.com/watch?v=HEtWHOiRKAg

---

*edit-

I kinda like the very top one as a pure fidelity increase on the OST, given it basically _is_ the OST, but with real instruments.
4th one's a great symphonic version (Distant World's was better, again, can't wait for them to release it on album),
5th one's amazing, but you'll probably disqualify it for the light vocals.
I love the 6th one, but I'm sure you guessed my weakness for good electric guitar riffs from my TBM additions, heh.
10th one will be great after I edit out the intro. Gets really nice towards the end.

Pick any of these you want to try out and I'll get them converted to PCM for you.

**Comment by Dizzy611** [comment]
Hopefully this doesn't sound too critical but this is one of my favorite parts of the soundtrack and it's a highly complex song.

First one is good, missing the airship-y sounds and the kind of "electronica" feel to parts of the instrumentation though, so I can't really say I recommend it. - Strong maybe.

Second one is decent but the levels seem a bit off. - No.

Third one is impressive as hell, still missing the "propeller"-y sounds though, though it has the synths (rather low in the mix though). - Maybe

Fourth one is really impressive and manages to simulate the propeller-y sound with an orchestra and the synth with what sounds like a grand piano, not *quite* the right sound but close enough to be impressive. - Strong maybe.

Fifth one... Flute's a bit too high and the intro sounds like it's being played on a kazoo. - Definite no.

Sixth one.. I was about to say it's great but after the intro they cut out half the instruments to focus on the guitars and drums, which sounds neat as a standalone thing on youtube but would be a bit odd 
ingame imo. - No leaning maybe.

Seventh one.. A very different interpretation but a pretty good one, seems to keep all the melodies and effects from the original intact in some form or other. - Strong maybe. Going to be hard to loop though given it's so short.

Eighth one... This one is a really great remix but I just don't feel it fits... It's too mellow, searching for friends in the game is kind of supposed to be your kick into gear after the previous depressing world theme, "I have a purpose now, lets kick some ass", etc. - Maybe.

Ninth one... YES. This is what I was looking for. Something that takes the original and, without missing any bits, improves on it. - Definite yes! (https://www.youtube.com/watch?v=dmHfcr858og for reference)

Tenth... No, way too slow. - No. Maybe with *extensive* editing.

Eleventh one... again YES! As good as the ninth, except this is the first one to get the synth and the left-to-right propeller effect perfect. - Definite yes!

Twelfth (and final from the first round of links) one... Nah, overuse of cheesy "chorus" orchestra effects. - No.

Distant worlds one sounds pretty amazing, agreed.
I'm picky, I know.

I'm actually a sucker for vocals when it comes to music to sit down and listen to, my problem with vocals is primarily when they're played on a loop in a game they get stuck in your head and annoy you really, really quickly.

In conclusion, I'd say see what you can do with:

1, 4, 6, 9, 10, 11. Special attention given to 9 and 11. 

There's a lot of these that would get my "Yes" if you could somehow add a "panning left to right propeller" effect to them that sounds like the original. I know it's a minor part of the track but it's an airship theme, it helps with the feel :P.

**Comment by edale2** [comment]
I got a long day at work today, so this is probably gonna wait until tomorrow to get worked on.

Oh, and Searching for Friends may start playing when you get the airship, but it's the worldmap theme, not just the airship theme.

If I could get an isolated chunk of the propeller sound you're talking about, I could add it to any track, the trick is finding the sound by itself.

**Comment by edale2** [comment]
I got a long day at work today, so this is probably gonna wait until tomorrow to get worked on.

Oh, and Searching for Friends may start playing when you get the airship, but it's the worldmap theme, not just the airship theme.

If I could get an isolated chunk of the propeller sound you're talking about, I could add it to any track, the trick is finding the sound by itself.

**Comment by edale2** [comment]
I got a long day at work today, so this is probably gonna wait until tomorrow to get worked on.

Oh, and Searching for Friends may start playing when you get the airship, but it's the worldmap theme, not just the airship theme.

If I could get an isolated chunk of the propeller sound you're talking about, I could add it to any track, the trick is finding the sound by itself.

**Comment by Dizzy611** [comment]
Slightly off-topic, but here's a prime example of "vocals that you hear over and over in a song in a game that get stuck in your head to the detriment of the game":

https://www.youtube.com/watch?v=1jwRkJV4CQY

By the time I got done with that game I was probably avoiding battles just so I didn't catch myself singing along to the battle theme.

**Comment by edale2** [comment]
Oh, I remember singing right along with the japanese lyrics I couldn't understand, but had memorized anyways, of the soundtrack from The World Ends With You (if you've never played it,. I HIGHLY recommend giving it a go, Nintendo DS game).

10th one... I don't really see what you're talking about with too slow, the intro and outro are a bit slow, but the rest of it seems fine. Sadly, it's not loopable for other reasons. Every verse has a different instrument setup, so it can't be looped, since no repetition of the same chord is the same.

Numbers 1, 4, 6, 9, and 11:
https://mega.nz/#!qRgy0ARI!nKDgeB8rcS3kwIeGTxqcusTtbcjlO76cm7P5kH9fWYs

**Comment by Dizzy611** [comment]
After extensive listening to the looped versions I think out of all of these I'd have to pick 4. 9 and 11 almost had it, but they both have this sample in it that sounds kind of like a squeaky door and it started to annoy me after a while.

It doesn't have the propeller effect quite right but it makes up for it by just being an excellent symphonic version of the track.

I tried ripping the samples from the FF3 ROM to see if I could find the one used for the propeller effect, no luck, I'm guessing they're using a single very simple noise sample repeated over and over, so unless I can find a way to play the SPC track but mute some of the channels it's unlikely I'm going to be able to isolate that bit.

**Comment by edale2** [comment]
I actually forgot to mention, but 11 actually wasn't a perfect loop, if you listen real closely, you'll notice that the "propeller" effect doesn't line up perfectly, but you have to be listening for it to notice that.

**Comment by edale2** [comment]
https://mega.nz/#!eRJTVIYI!DLFYZc5zk1SMqyFFsYUY7Y2DCuL6ln6AxeceMiAQy0A

_edale2 whistles innocently._

On a _completely_ unrelated note: http://www.jcomcp.plus.com/jcomspc.html for an SPC player that lets you selectively mute some of the sound channels.

**Comment by edale2** [comment]
[midi.zip](https://github.com/Insidious611/DancingMadFF6/files/1462757/midi.zip)

Look (listen) at some of the interesting effects I was able to get by isolating that track in an SPC to MIDI converter, and playing with the instrument settings. Sadly, none of the available options sounded anything like the track in-game.

*edit- I was hoping that if I could get that channel as an actual midi, I'd be able to use some midi software (no clue which one, I'd have to google one up) to change the sound up to better match whatever track I was adding it to. No dice, so I can only add the effect as-is.

Speaking of that, did you want me to try adding it to any of those versions of the tracks? You said lacking that effect was the main problem for some of them.

**Comment by Dizzy611** [comment]
If you could see how it sounds with that propeller effect added to 1, 3, 4, and 5 I'd appreciate it if you have time. If you want to save time, 4 is probably the one I'm going with so far.

**Comment by edale2** [comment]
Oh my. I saw 5 was part of an actual album, so I checked it out, and it's not for sale anymore anywhere I can find, and can be listened to in it's entirety for free from several streaming sites.

So I feel no remorse in saying I found a FLAC copy of the entire album, and I'm seriously considering converting the entire thing, it's fantastic. 22 tracks, includes every character's theme songs, and even includes "Balance is Restored", the ending theme... which means if you use these tracks for the game, the ending where it plays all the character themes, it'll be the same ones you heard throughout the rest of the game!

**Comment by edale2** [comment]
https://mega.nz/#!HUJD0ZoT!7E39POt0oJGeLhSUNisp67sP4KOlDOn8cIDFMSuMGXc
Tracks 1, 3, 4, and 5. All with the effect added.

**Comment by edale2** [comment]
Oh, and if you think the effect needs to be louder/quieter in a track, say the word and I can do it. as-is the propeller effect might be a bit too loud in some of them.

**Comment by Dizzy611** [comment]
Just gave a listen, yeah I agree, try them all with the propeller bit a bit softer in the mix.

**Comment by edale2** [comment]
https://mega.nz/#!yZQ0xbYL!ALRXZwhCK1FS1KiZOoHv7cN0dQ0PPepJ_-VJONDaEs0
Try those ones, should be better.

**Comment by edale2** [comment]
https://www.youtube.com/watch?v=-q41N_jFdhU
Just found that one, it's pretty good, lemme know if you wanna get it converted (and the propeller effect added)

**Comment by Dizzy611** [comment]
I think I'm actually going to go with 3-3. It's grown on me, and with the propeller sound a bit lower in the mix but still there it sounds pretty good.

**Comment by Dizzy611** [comment]
ff3-73-3 uploaded to the sources folder under OTH. Obviously not part of the installer yet.

**Comment by edale2** [comment]
It's useless for converting because of how the songs are mashed up together (songs are actually mashed together rather than just playing one after the other), but this mashup of Searching For Friends with FFIX's You Are Not Alone is AMAZING

https://www.youtube.com/watch?v=92Fc1tTP-dU&list=PLhsnNwoLVWWO_dUKXtlyFgG77N5Gw6zpj&index=57

---

And from the same person: https://www.youtube.com/watch?v=ZebHgNuoVw0&index=9&list=PLhsnNwoLVWWO_dUKXtlyFgG77N5Gw6zpj

How would you feel about this being converted as an OTH track 1 (Prelude, IIRC it only plays on the load screen and the end screen)?

It's an original composition in the style of the various Final Fantasy Preludes/Crystal themes. Ordinarily I'd say it wouldn't work too well in-game, but given the only locations the song plays... It'd actually work nicely, I think.

**Comment by Dizzy611** [comment]
Closing as this has been officially added into the XML file for installer xmlification, and is in the working folder (locally) which will be uploaded to the mirrors when I'm ready to release the new installer. So, basically, any further work on this will be done as a matter of course for the rest of the work being done.
